skabase/SKABaseDevice/SKABaseDevice.py

```diff
@@ -36,9 +36,7 @@
 import logging
 import logging.handlers
 from logging.handlers import SysLogHandler
-
-# Initialize logging
-logging.basicConfig()
+from logging.handlers import WatchedFileHandler
```

Comment:
This breaks Windows portability. Was there an issue with using RotatingFileHandler or similar?

Reply:
You're right. We're not aiming at Windows compatibility now, and we do not have portability tests in place. Nevertheless I value portability as a quality of the software and I'd encourage the adoption of portable alternatives whenever possible. Even though portability is not a requirement now, it is certainly an important quality for a system that needs to live for 50 years.

Reply:
Replaced WatchedFileHandler with RotatingFileHandler.